### PR TITLE
Dataframes exercise

### LINKED ISSUE #22: More Exercises
We should have 2 more short exercises (one "write some not entirely trivial code in Julia" and one "do some plots or dataframes or both").

I'll have a look at this today, but if anyone has suggestions I am open to them for topics.

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Fairly straight forward Data Frames exercise, that should fit into the time we have.

A sample answer is also provided as a notebook.

Closes #22

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
The natural extension of this is to then use an ML library to train a model and sort signal from background, but as we don't have time for Julia ML packages that won't work this time around.

**Comment by Moelf** [comment]
Missing from toc file?

**Comment by graeme-a-stewart** [comment]
Oops, yes - rebase went wrong, I guess!

File: julia-intro/docs/exercises/dataframes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# DataFrames Exercise
+
+You can do this exercise in a notebook or as a script (or even in the REPL!).
+
+## Load and Clean-up a DataFrame
+
+Download the reduced Higgs ML dataset from [here](https://github.com/JuliaHEP/JuliaHEP-2023/raw/main/julia-intro/docs/assets/atlas-higgs-challenge-2014-v2-reduced.csv).
```
Comment (Moelf) [inline julia-intro/docs/exercises/dataframes.md:7]: I think they should have this either way -- clone repo or opening in Binder

**Review by Moelf — APPROVED** [review]

**Pull request merged**